packages/components/src/components.ts

```diff
@@ -24,6 +24,11 @@ import HdsDialogPrimitiveOverlay from './components/hds/dialog-primitive/overlay
 import HdsDialogPrimitiveWrapper from './components/hds/dialog-primitive/wrapper.ts';
 import HdsDisclosurePrimitive from './components/hds/disclosure-primitive/index.ts';
 import HdsDismissButton from './components/hds/dismiss-button/index.ts';
+import HdsFlyoutBody from './components/hds/flyout/body.ts';
+import HdsFlyoutDescription from './components/hds/flyout/description.ts';
+import HdsFlyoutFooter from './components/hds/flyout/footer.ts';
+import HdsFlyoutHeader from './components/hds/flyout/header.ts';
+import HdsFlyout from './components/hds/flyout/index.ts';
```

Comment:
note for posterity: we generally don't expose subcomponents here, only the main component; however, there are currently instances in HCP where Flyout subcomponents are used in TypeScript

Reply:
I added a note to discuss this in eng sync tomorrow... until then, I'll merge this as is?

Reply:
Yes, also because these sub-components are already deprecated: https://helios.hashicorp.design/components/flyout?tab=version%20history#deprecated